src/components/Home/DurationAttributePanel.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import {
+  SceneComponentProps,
+  SceneFlexLayout,
+  sceneGraph,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneQueryRunner,
+} from '@grafana/scenes';
+import { LoadingState } from '@grafana/data';
+import { explorationDS } from 'utils/shared';
+import { LoadingStateScene } from 'components/states/LoadingState/LoadingStateScene';
+import { MINI_PANEL_HEIGHT } from 'components/Explore/TracesByService/TracesByServiceScene';
+import { yBucketToDuration } from 'components/Explore/panels/histogram';
+import { AttributePanel, SkeletonComponent } from './AttributePanel';
+import { getErrorMessage, getNoDataMessage } from 'utils/utils';
+import { AttributePanelScene } from './AttributePanelScene';
+
+export interface DurationAttributePanelState extends SceneObjectState {
+  panel?: SceneFlexLayout;
+}
+
+export class DurationAttributePanel extends SceneObjectBase<DurationAttributePanelState> {
```

Comment:
Looks very similar to `AttributePanel` - would it make sense to extend the class to reuse some code? 🤔